pkg/util/globalerror/grpc.go

```diff
@@ -24,10 +28,12 @@ func WrapGRPCErrorWithContextError(err error) error {
 	if stat, ok := grpcutil.ErrorToStatus(err); ok {
 		switch stat.Code() {
 		case codes.Canceled:
-			return &ErrorWithStatus{
-				UnderlyingErr: err,
-				Status:        stat,
-				ctxErr:        context.Canceled,
+			if stat.Message() != grpcClientConnectionIsClosingErr {
```

Comment:
It's a bummer that we depend on the error message not changing. Perhaps we could compare the err to `grpc.ErrClientConnClosing`? Or convert that to a status, then compare the error message?

Reply:
Hi @jhalterman 
If we use `grpc.ErrClientConnClosing` lint will complain because that error is deprecated. And as far as I know `grpc.ErrClientConnClosing` is the only exported occurrence of the error message we are looking for.